service/src/test/scala/ch/epfl/bluebrain/nexus/commons/es/server/embed/ElasticServer.scala

```diff
@@ -0,0 +1,89 @@
+package ch.epfl.bluebrain.nexus.commons.es.server.embed
+
+import java.nio.file.Files
+import java.util.Arrays._
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.Uri
+import akka.testkit.TestKit
+import ch.epfl.bluebrain.nexus.commons.es.server.embed.ElasticServer.MyNode
+import ch.epfl.bluebrain.nexus.commons.test.Randomness
+import org.apache.commons.io.FileUtils
+import org.elasticsearch.common.settings.Settings
+import org.elasticsearch.index.reindex.ReindexPlugin
+import org.elasticsearch.node.Node
+import org.elasticsearch.painless.PainlessPlugin
+import org.elasticsearch.plugins.Plugin
+import org.elasticsearch.transport.Netty4Plugin
+import org.scalatest.wordspec.AnyWordSpecLike
+import org.scalatest.BeforeAndAfterAll
+
+import scala.jdk.CollectionConverters._
+import scala.util.Try
+
+// $COVERAGE-OFF$
+abstract class ElasticServer
```

Comment:
We probably don't need this embedding service anymore. But we can keep it until we change the tests